src/Persistence_representations/test/persistence_heat_maps_test.cpp

```diff
@@ -78,7 +78,7 @@ BOOST_AUTO_TEST_CASE(check_compute_percentage_of_active_of_heat_maps) {
   to_compute_percentage_of_active.push_back(&q);
   to_compute_percentage_of_active.push_back(&r);
   Persistence_heat_maps<constant_scaling_function> percentage_of_active;
-  percentage_of_active.compute_percentage_of_active(to_compute_percentage_of_active, 0.1);
```

Comment:
My first reaction would be to think that this was supposed to be 0.1 times something, but 0 seems consistent with the expected data/template_percentage_of_active_of_heat_maps. I don't really understand why `compute_percentage_of_active` is named this way, I don't see any percentage in it...

Reply:
It seems to indirectly compute the percentage by counting the number of maps with particular values but without dividing by the number of maps. I think it was supposed to be a percentage first (therefore a cutoff of 0.1), but ended up just counting and the test was not corrected.